unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/ChannelLimitReachedWindowController.cs

```diff
@@ -0,0 +1,35 @@
+namespace DCL.Chat.HUD
+{
+    public class ChannelLimitReachedWindowController : IHUD
```

Comment:
Following the discussing about the use of the `HUDController`/`HUDFactory` classes to instantiate huds, we should stop using them and take advantage of our new plugin system. We have currently some huds working through a system and other ones through other system. In this case I think we could have integrated this hud through the plugin system in the same way we do for the JoinChannel HUD (`JoinChannelComponentController`) and not to make it dependant of the other features like the taskbar.

Anyway it is not something that we have to fix in this PR, but for start thinking about this and reach/implement a common way together for the near future 👍

Reply:
agree, done!